### DISCUSSION TITLE
add another design

### DISCUSSION THREAD

**Discussion by Roy-debug-hub** [ansys/pyaedt#5435] [Q&A]
How to import another edb as the second design in a project that already has a design.

**Comment by svandenb-dev** [comment]
Hi @Roy-debug-hub 
I assume you want to merge a design onto another one. In PyEDB we have several methods
You can check here:
https://edb.docs.pyansys.com/version/stable/api/_autosummary/pyedb.dotnet.edb_core.stackup.Stackup.place_in_layout.html
or this one which I would recommend
https://edb.docs.pyansys.com/version/stable/api/_autosummary/pyedb.dotnet.edb_core.stackup.Stackup.place_in_layout_3d_placement.html
Using 3D placement provides easier handling like flipping and you can also assign arbitrary elevation. 
You can also evaluation the vector to align the merge design:
https://edb.docs.pyansys.com/version/stable/api/_autosummary/pyedb.dotnet.edb_core.components.Components.get_component_placement_vector.html

Here is a code snippet as an example:

`targetfolder = r'C:\Temp\package_on_board'

targetfile1 = os.path.join(targetfolder, 'board.aedb')
targetfile2 = os.path.join(targetfolder, 'package.aedb')
targetfile2_merged = os.path.join(targetfolder, 'package_merged.aedb')


edb1 = Edb(edbpath=targetfile1,  edbversion="2021.2")
edb2 = Edb(edbpath=targetfile2, edbversion="2021.2")

hosting_cmp = edb1.core_components.get_component_by_name("U100")
mounted_cmp = edb2.core_components.get_component_by_name("BGA")

res, vector, rotation, solder_ball_height = edb1.core_components.get_component_placement_vector(
                                                    mounted_component=mounted_cmp,
                                                    hosting_component=hosting_cmp,
                                                    mounted_component_pin1="A12",
                                                    mounted_component_pin2="A14",
                                                   hosting_component_pin1="A12",
                                                   hosting_component_pin2="A14")
edb2.stackup.place_in_layout_3d_placement(edb1, angle=0.0, offset_x=49e-3,
                                  offset_y=42e-3, flipped_stackup=False, place_on_top=False)

edb1.close_edb()
edb2.save_edb_as(targetfile2_merged)
edb2.close_edb()`

**Reply by Roy-debug-hub** [reply]
Thank you for your reply. I don't want to merge two edb files. I want to import another edb as the second design in the project that already has a design in hfss3dlayout.
